rest-api-cloud-client/src/test/java/com/stnetix/ariaddna/externalcloudapi/test/IAbstractCloudTest.java

```diff
@@ -0,0 +1,130 @@
+package com.stnetix.ariaddna.externalcloudapi.test;
+
+import com.google.gson.JsonObject;
+import com.stnetix.ariaddna.externalcloudapi.cloudinterface.IAbstractCloud;
+import com.stnetix.ariaddna.externalcloudapi.implementation.YandexDisk;
+import okhttp3.*;
+import org.junit.*;
+
+import java.io.*;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class IAbstractCloudTest {
+    private static IAbstractCloud cloud;
+    private static JsonObject result;
+    private static URL imgUrl1, imgUrl2, bookUrl;
+    private static File root, docs, books, pics;
+    private static List<File> files = new ArrayList<>();
+
+    @BeforeClass
+    public static void setUp() throws Exception{
+        imgUrl1 = new URL("http://st.gde-fon.com/wallpapers_original/wallpapers/300884_(www.Gde-Fon.com).jpg");
```

Comment:
Don't use external resource in Unit Test, because they may not be available and this test are fail.